go.mod

```diff
@@ -4,9 +4,11 @@ go 1.19
 
 require (
 	github.com/Masterminds/semver v1.5.0
+	github.com/go-git/go-git/v5 v5.5.2
```

Comment:
Hey no problem adding dependencies but I don't see any usage of these. Why did you add them?

Reply:
I didn't add anything. I think the tooling just reordered the dependencies by alphabetic order (see the removes lines below).

Reply:
Ok my bad!